Ago-Dic-2020/rodriguez-martinez-diego-jose/Practica 1/matrix.py

```diff
@@ -0,0 +1,27 @@
+class Matrix:
+    matrix_List = []
```

Comment:
No necesitas declarar el atributo aquí, ya que esto sería un atributo de clase y no de la instancia. Checa el siguiente artículo 😉  -> https://www.digitalocean.com/community/tutorials/understanding-class-and-instance-variables-in-python-3

Reply:
Ahh, ya. Crei que era necesario crear una variable de clase para poder accederla en los otros metodos, no sabia acerca de las variables de instancia.